routing/missioncontrol.go

```diff
@@ -308,6 +319,9 @@ func (m *MissionControl) SetConfig(cfg *MissionControlConfig) error {
 	m.state.minFailureRelaxInterval = cfg.MinFailureRelaxInterval
 	m.estimator = cfg.Estimator
 
+	// Callback function to update the estimator value on main cfg.
```

Comment:
I thought it would be better for the understanding, without that the comment would be something like "Execute the callback function." ... or maybe it does not need any comment!